modules/xmpp/JingleSessionPC.js

```diff
@@ -2492,22 +2390,16 @@ export default class JingleSessionPC extends JingleSession {
 
                 return;
             }
-            const oldLocalSDP = tpc.localDescription.sdp;
             const operationPromise
                 = isRemove
                     ? tpc.removeTrackFromPc(track)
                     : tpc.addTrackToPc(track);
 
             operationPromise
                 .then(shouldRenegotiate => {
-                    if (shouldRenegotiate && oldLocalSDP && tpc.remoteDescription.sdp) {
+                    if (shouldRenegotiate) {
                         this._renegotiate()
-                            .then(() => {
-                                // The results are ignored, as this check failure is not enough to fail the whole
-                                // operation. It will log an error inside for plan-b.
-                                !this.usesUnifiedPlan && this._verifyNoSSRCChanged(operationName, new SDP(oldLocalSDP));
-                                finishedCallback();
-                            });
+                            .then(() => finishedCallback());
```

Comment:
do you need the extra arrow function? you can pass the callback straigth to then.